packages/iris-grid/src/ColumnStatistics.scss

```diff
@@ -23,8 +24,6 @@
   color: $gray-400;
   font-weight: normal;
   user-select: none;
-  text-overflow: ellipsis;
-  overflow: hidden;
```

Comment:
Why'd you remove this truncation?

Reply:
I don't think we actually want to hide the column type.